client/homebrew/brewRenderer/brewRenderer.jsx

```diff
@@ -149,13 +171,12 @@ const BrewRenderer = createClass({
 	renderPage : function(pageText, index){
 		let cleanPageText = this.sanitizeScriptTags(pageText);
 		if(this.props.renderer == 'legacy')
-			return <div className='phb page' id={`p${index + 1}`} dangerouslySetInnerHTML={{ __html: MarkdownLegacy.render(cleanPageText) }} key={index} />;
+			return <div className='pages' ref='pages' lang={`${this.props.lang || 'en'}`}><div className='phb page' id={`p${index + 1}`} dangerouslySetInnerHTML={{ __html: MarkdownLegacy.render(cleanPageText) }} key={index} /></div>;
```

Comment:
This now renders a separate `pages` div around each individual page, rather than one large `pages` object that contains all pages. Can you help me understand the reasoning for this?

Reply:
That is a mistake; I was so focused on V3 that I missed that change to Legacy. I'll dig into it and correct.